build.gradle

```diff
@@ -266,6 +261,19 @@ test {
             logger.lifecycle("Test: " + descriptor + " produced standard out/err: " + event.message )
         }
     }
+
+    testLogging {
```

Comment:
This forces the test output to always include notification for every skipped or failed test, as well as a stack trace if necessary.  It previously wasn't clear when running single tests or not if they were skipped or if they passed.  Now it's obvious. 

It also adds a line after the whole test suite is run that includes the total number of tests run and the count of succeeded/skipped/failed because that wasn't always clear.